docs/variables/valorant.md

```diff
@@ -0,0 +1,153 @@
+---
+id: valorant
+---
+
+# $(valorant)
+
+Returns a range of metadata around a broadcaster's [**Riot Games**](https://www.riotgames.com/en) account, in relation to the game VALORANT, that can be accessed via multiple different variable members for each unique piece of information.
+
+:::caution Authorization required!
+
+This variable will only function if a broadcaster has authorized and connected a Riot Games account within Fossabot's dashboard under the **Integrations** tab.
+
+:::
+
+* **Supported Game Modes**
+  * `competitive`
+  * `deathmatch`
+  * `escalation`
+  * `premier`
+  * `replication`
+  * `spikerush`
+  * `swiftplay`
+  * `unrated`
+
+* **Special Modifiers**
+  * `include_custom` *(adds custom games to be considered)*
+  * `custom_only` *(limits the considered games to **custom games only**)*

```

Comment:
The league one includes a `-` between the name and description, should it be formatted the same way?

Reply:
sure, lets do it